src/frontend/app/contact/src/app/globals.css

```diff
@@ -0,0 +1,107 @@
+:root {
```

Comment:
do we want the nextjs and vercel logos? They seem to be used in a page that is temporary.

Reply:
[5e41abb](https://github.com/numerique-gouv/people/pull/3/commits/5e41abbf807ebad5a773b013a2447c1b52a04236)